README.md

```diff
@@ -20,3 +20,27 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+## Answer 
+
+In our permutationSort function we have two main steps generating all the permutations and checking if the permutation is sorted. 
+1. generatePermutations: We know that there are $n!$ possible permutations for an array of size $n$. The function uses recursions to swap the elements and generate each permrutation. Since every possible permutation is generated the number of recursive calls the reach the base case is $n!$. Each time a permutation is created we add it to the results which is linear and does not change the dominant $n!$ complexity.
+2. isSorted: This function checks each element of the permutation to determine if the array is in ascending order. For each permutation, isSorted has to go through $n$ elements. This however does not effect the time as we are only interested in the dominant term as $n$ increases in size.
+
+This leaves us with the overall time complexity of $O(n!)$. 
```

Comment:
What do you do for each permutation?

Reply:
No, but how much work is that, and what effect does that have on the overall complexity?

Reply:
Oh ok, We do loop through each permutation so that would increase our overall complexity.